asmcli/commands/install.sh

```diff
@@ -192,6 +192,7 @@ install_control_plane() {
   if [[ "${MANAGED}" -eq 1 ]]; then
     install_managed_control_plane
   else
+    label_istio_namespace
```

Comment:
@zerobfd should this call be outside the surrounding if/else ? (In order to apply it for both in-cluster and MCP.)

Reply:
yep

Reply:
done